app/controllers/posts_controller.rb

```diff
@@ -62,6 +66,19 @@ def users_follow
   end
 
   def posts_params
-    params[:post].permit(:title, :body, :tag_list, :slug, :images)
+    params[:post].permit(:title, :body, :tag_list, :slug)
+  end
+
+  def temporarily_id
+    @temporarily_id || params[:post][:temporarily_id]
+  end
+
+  def generate_post_tmp_id
```

Comment:
We have Post model.
When using `medium-editor` we upload images before Post was created (it is `medium-editor`'s behavior of uploading images). So we need to determine which images were uploaded for this Post.

It means that we create random temporarily id for Post before it was saved and persisted. This post id is saved when we upload image and save Post::Picture. Then when we save Post this tmp id is being passed as parameter and used to find pictures for this post.

When we edit Post this parameter is set to a post's id.